flag.go

```diff
@@ -742,7 +760,7 @@ func (f *FlagSet) FlagUsagesWrapped(cols int) string {
 
 		line += usage
 		if !flag.defaultIsZeroValue() {
-			if flag.Value.Type() == "string" {
```

Comment:
Hm... overlooked this part; this will be tricky, because we're _removing_ the `Type()` from the `Flag.Value` - that will likely break people as they would now have to implement code similar to what's in the PR to assert that the flag Value implements the `TypedValue`; https://github.com/spf13/pflag/blob/10438578954bba2527fe5cae3684d4532b064bbe/flag.go#L192-L196

Reply:
I guess the (philosophical) question is whether the breaking change is _introducing_ this method on the `Value` type (thus breaking the promise of being a drop-in replacement for stdlib `flag`), or _removing_ it. If one takes the former stance, then this is a bug fix; with the latter, it's a major version bump. (So after all the flak for 1.0.8, it turns out it's not even clear cut what a breaking change _is_. Who'da thunk? :wink:) 

My guess is that there are quite a few instances of people implementing Type because our interface requires it from them, but much fewer (if anyone) actually using the method for anything on their end.